test/unit/builtins/math.spec.ts

```diff
@@ -24,3 +24,13 @@ test.each(["E", "LN10", "LN2", "LOG10E", "LOG2E", "SQRT1_2", "SQRT2"])("Math.%s"
         expect(builder.getLuaExecutionResult()).toBeCloseTo(builder.getJsExecutionResult());
     });
 });
+
+const expectMathAtan2: util.TapCallback = builder => expect(builder.getMainLuaCodeChunk()).toMatch("math.atan2");
+const expectMathAtan: util.TapCallback = builder => expect(builder.getMainLuaCodeChunk()).toContain("math.atan");
```

Comment:
Is there a reason to use `toMatch` in the first check and `toContain` in the second?

Also, wouldn't `.toContain("math.atan")` match `atan2` as well? If so matched value should include `(`.